modules/ols-0-2-1-release-notes.adoc

```diff
@@ -22,4 +22,6 @@ The following enhancements have been made with {ols-official} 0.2.1:
 
 * This release adds the ability for users to provide their own TLS certificates for communication between the {ols-long} console and the {ols-long} service.
 
-* With this release, the {ols-long} User Interface displays an email link that can be used to contact the {ols-official} Development Team.
\ No newline at end of file
+* With this release, the {ols-long} User Interface displays an email link that can be used to contact the {ols-official} Development Team.
+
+* This release introduces the ability to attach a YAML file to a question you ask on any page accessible from *Storage*, *Compute* or *Builds* in the {ocp-product-title} web console.
```

Comment:
Small nit - I'd just use the Oxford comma per the [ISG](https://www.ibm.com/docs/en/ibm-style?topic=punctuation-commas#commas-between-items-in-a-series):
```suggestion
* This release introduces the ability to attach a YAML file to a question you ask on any page accessible from *Storage*, *Compute*, or *Builds* in the {ocp-product-title} web console.
```

Reply:
done